Automattic/simplenote-ios
# Case insensitive support for Tags Operator

### Fix
In this PR we're making the (localized) Tag Search Operator case insensitive, so that translations that might be capitalized don't break.

@diegoreymendez May I trouble you with this one?
Thanks in advance!

Closes #734

### Test
1. Log into your account
2. Add a new Note with a **RANDOM** name
3. Set the Simulator's Application Language to German
4. Relaunch the app
5. Press over the Search Bar
6. Type **RANDOM** and select the "Filter by Tag" row (`Schlagwort:RANDOM`)

- [x] Verify your note shows up

### Release
`RELEASE-NOTES.txt` was updated in 02e8692 with:
 
> Fixed a bug that prevented the Tag Search Operator not to properly filter in non english languages.
